### PR TITLE
Release 5.4

### PR DESCRIPTION
**Pull request by Siedlerchr**

### CODE REVIEW

File: _posts/2021-12-23-JabRef5-4.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+bg: jabref-font
+color: white
+---
+
+We are proud to announce the release of the next milestone of JabRef, this time JabRef 5.4, just in time for the holidays!
+It's been roughly half a year since the previous release of 5.3, and we are happy to announce that despite the still ongoging Covid-19 waves we were all fortunately able to continue working on JabRef.
+
+## Notable Changes and New Features
+
+Thanks to this year's Google Summer of Code, JabRef now as a PDF-fulltext search and improved metadata support. Search across libraries and in your linked pdf files.
```
Comment (btut) [inline _posts/2021-12-23-JabRef5-4.md:14]: *has

**Review by btut — APPROVED** [review]
LGTM, except for that small typo.

File: _posts/2021-12-23-JabRef5-4.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title:  🎄 JabRef 5.4 Release 🎄
+id: jabref-5-4
+author: "[JabRef Maintainers](https://github.com/JabRef/jabref/blob/main/MAINTAINERS)"
+bg: jabref-font
+color: white
+---
+
+We are proud to announce the release of the next milestone of JabRef, this time JabRef 5.4, just in time for the holidays!
+It's been roughly half a year since the previous release of 5.3, and we are happy to announce that despite the still ongoging Covid-19 waves we were all fortunately able to continue working on JabRef.
```
Comment (mlep) [inline _posts/2021-12-23-JabRef5-4.md:10]: ongoging --> ongoing

Comment (Siedlerchr) [reply _posts/2021-12-23-JabRef5-4.md:10]: Feel free to directly edit it!

File: _posts/2021-12-23-JabRef5-4.md (hunk 1 above)
Comment (mlep) [inline _posts/2021-12-23-JabRef5-4.md:14]: Missing subject and verb in "Search across libraries and in your linked pdf files."?

File: _posts/2021-12-23-JabRef5-4.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Notable Changes and New Features
+
+Thanks to this year's Google Summer of Code, JabRef now as a PDF-fulltext search and improved metadata support. Search across libraries and in your linked pdf files.
+Search results across several libraries can now also be shown again in a new window. It's also possible to keep the search string when switching between libraries.
+Another major improvement is the metadata extraction and writing for PDFs. When generating an entry from a PDF, JabRef now checks different sources and provides an improved merge dialog to combine the extracted metadata.
+
+Further work focused on improving the user experience in the GUI. For example, we added an icon picker to the Groups dialog to easily search for a nice group icon.
```
Comment (mlep) [inline _posts/2021-12-23-JabRef5-4.md:18]: nice -> suitable ?

File: _posts/2021-12-23-JabRef5-4.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+We also added the possibilty to search all avaiable citation styles together with a sample preview.
```
Comment (mlep) [inline _posts/2021-12-23-JabRef5-4.md:19]: avaiable  -> available

Comment (mlep) [inline _posts/2021-12-23-JabRef5-4.md:19]: possibilty  -> possibility

File: _posts/2021-12-23-JabRef5-4.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Despite this new features, we also fixed many bugs. Some of them were related to import/export and the handling of linked files.
```
Comment (mlep) [inline _posts/2021-12-23-JabRef5-4.md:21]: Despite this --> Additionally to these

File: _posts/2021-12-23-JabRef5-4.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+For a complete list of all our changes, take a look at the [Changelog](https://github.com/JabRef/jabref/blob/main/CHANGELOG.md) since our last official release!
+
+**You can get it from [FOSShub](https://www.fosshub.com/JabRef.html).**
+
+In case you notice any bugs or problems, please check our [bug tracker](https://github.com/JabRef/jabref/issues) and consider submitting a new bug report if nobody else has been reporting the failure before.
+
+## Special Thanks
+
+The [JabRef Maintainers](https://github.com/JabRef/jabref/blob/main/MAINTAINERS) would like to give special thanks to all new and old contributors. We are delighted to see so many new contributors opening pull requests and making translations. As we all work on JabRef in our free time, we feel honored that people share our enthusiasm for free software and work together to make research easier.
```
Comment (mlep) [inline _posts/2021-12-23-JabRef5-4.md:30]: Since people making translations are thanked, maybe the new Korean localization should be mentioned above.

Comment (Siedlerchr) [reply _posts/2021-12-23-JabRef5-4.md:30]: Indeed! Good idea

**Pull request merged**